change/@ni-nimble-components-6e819e6f-2633-43f7-8e29-c38c7850ee59.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Add BodyPlus1 font token",
```

Fixing the bodyPlus1Emphasized token name makes this major

```suggestion
  "type": "major",
  "comment": "Breaking change: Renamed bodyEmphasizedPlus1 to bodyPlus1Emphasized to align with token conventions. Also, added BodyPlus1 font token",
```